src/v2/Apps/Auctions/Components/__tests__/TrendingLotsRail.jest.tsx

```diff
@@ -64,6 +64,16 @@ describe("TrendingLotsRail", () => {
     expect(wrapper.find("FillwidthItem")).toBeDefined()
   })
 
+  it("skips closed lots", () => {
+    const wrapper = getWrapper({
+      SaleArtworksConnection: () => ({
+        edges: [{ node: { sale: { isClosed: true } } }],
+      }),
+    })
+
+    expect(wrapper.html()).toContain("No Works To Show")
```

Comment:
This errors and is very confusing! When I log out the node inside the component it does NOT match the mock data there in the `getWrapper` call - why is that? 🤔

Reply:
And @damassi note that we also tried `Sale: () => ({ isClosed: true })` which is what I'd typically do and didn't work either